packages/mui-base/src/Switch/Switch.tsx

```diff
@@ -126,10 +133,10 @@ const Switch = React.forwardRef(function Switch<RootComponentType extends React.
   });
 
   return (
-    <Root {...rootProps}>
+    <Root {...rootProps} sx={{ outline: 'none' }}>
       <Track {...trackProps} />
       <Thumb {...thumbProps} />
-      <Input {...inputProps} />
+      <Input {...inputProps} hidden type="hidden" />
```

Comment:
There's no need to have both `hidden` and `type="hidden"`. Let's remove the `hidden` and get the `type` from `inputProps`.

```suggestion
      <Input {...inputProps} />
```

Reply:
Okay, I will change it. I just thought that since the hook is also used for `Radio` and `Checkbox` and the hidden input is special for `Switch`, the hidden type should be set on `Switch.tsx` not on the hook.

Reply:
I think we shouldn't add hidden to `getInputProps` since, as catched by the unit tests, it causes other component to have hidden input as well (Radio and Checkbox). I'm moving it back to `<Input {...inputProps} hidden />`

Reply:
> I just thought that since the hook is also used for Radio and Checkbox

Ah, you mean the Joy components? They use `useSwitch` only because we don't have proper `useCheckbox` and `useRadioButton` just yet. They should be refactored once we create these hooks. For now, let's do whatever changes are neccessary in Base UI as this affects the public API on which developers will depend on. If we need to make some hacks in the implementation of Joy components because of this, so be it - their implementation will change anyway when we create proper hooks for them (cc @siriwatknp)